extensions/amp-story/0.1/amp-story.js

```diff
@@ -323,7 +330,7 @@ export class AmpStory extends AMP.BaseElement {
 
     // first page is not counted as part of the progress
     // TODO(alanorozco): decouple this using NavigationState
-    this.systemLayer_.updateProgressBar(pageIndex, this.getPageCount() - 1);
+    this.progressBar_.setActivePageIndex(pageIndex);
```

Comment:
I don't think `AmpStory` should have knowledge of the progress bar object. I think this would be better:

`this.systemLayer_.setActivePageIndex()`

and that way you won't need to expose `ProgressBar`

Reply:
Done!